### PR TITLE
feat(binary): combine query-service and frontend

### PR DESCRIPTION
**Pull request by grandwizard28**
### Summary

This will combine query-service and frontend into a single binary called signoz. 

#### Related Issues / PR's

This is a duplicate of https://github.com/SigNoz/signoz/pull/5710

### CODE REVIEW

File: conf/defaults.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  dsn: tcp://localhost:9000
+  cluster: cluster
+  prometheus_config_path: /etc/signoz/config/prometheus.yml
+  max_idle_connections:
+  max_open_connections:
+  dial_timeout:
+  optimize_read_in_order_regex:
+  max_execution_time_leaf:
+  timeout_before_checking_execution_speed:
+  max_bytes_to_read:
```
Comment (grandwizard28) [inline conf/defaults.yaml:104]: Need better names here @srikanthccv!
```suggestion
storage:
  provider: clickhouse
  dsn: tcp://localhost:9000
  cluster: cluster
  prometheus_config_path: /etc/signoz/config/prometheus.yml
  max_idle_connections:
  max_open_connections:
  dial_timeout:
  optimize_read_in_order_regex:
  max_execution_time_leaf:
  timeout_before_checking_execution_speed:
  max_bytes_to_read:
```

Comment (grandwizard28) [reply conf/defaults.yaml:104]: prometheus:
   remote_read:
      config:
      url:

File: conf/defaults.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+#----------------------------------------------
+database:
+  # Supported options are sqlite
+  provider: sqlite
+  # Path to the sqlite database
+  path: /var/lib/signoz
+#----------------------------------------------
+# Storage
+#----------------------------------------------
+storage:
```
Comment (srikanthccv) [inline conf/defaults.yaml:93]: We should probably keep each provider options independently, like

```
storage:
	provider: clickhouse
	clickhouse:
		...
```

```
database:
	provider: sqlite
	sqlite:
		...
```

File: conf/defaults.yaml (hunk 1 above)
Comment (srikanthccv) [inline conf/defaults.yaml:104]: Let's break these into two settings 1. client settings and 2. query settings.

**Review by srikanthccv — COMMENTED** [review]
Two observations

**Pull request closed without merging**